openbb_terminal/terminal_controller.py

```diff
@@ -413,6 +413,7 @@ def call_stocks(self, _):
 
         self.queue = self.load_class(StocksController, self.queue)
 
+
```

Comment:
Why are we adding an empty new line here?

Reply:
This empty new line may have been added by mistake.